env/shared.env

```diff
@@ -13,3 +12,6 @@ NGINX_PORT=8063
 APISIX_PORT=8065
 KEYCLOAK_PORT=8066
 KEYCLOAK_SSL_PORT=8067
+
+# Run all containers by default
+COMPOSE_PROFILES=backend,frontend,keycloak,apisix
```

Comment:
I don't actually think this will work. For `docker compose` to recognize `COMPOSE_PROFILES`, they need to either be in your shell env or in a `.env` file adjacent to the compose file.

So that's a little awkward. Otherwise everything looks good.

Reply:
removed, and updated the README's